src/FieldWarning/Assets/Util/ConfigReader.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using System;
+/**
+ * Copyright (c) 2017-present, PFW Contributors.
```

Comment:
The thing that automatically resolves usings does this sometimes..

Reply:
Fixed!